build/azure-pipelines/build-common.yml

```diff
@@ -7,8 +7,11 @@ parameters:
   default: 
 - name: ArtifactName
   type: string
+- name: VersionTag
+  type: string

```

Comment:
you're missing the update to build-product.yml to pass the value

Reply:
I missed a git add!